tests/nativeapp/test_manager.py

```diff
@@ -28,8 +34,16 @@
                 - src: app/streamlit/*.py
                   dest: ui/
 
+            application:
+                name: myapp
+                role: app_role
+                warehouse: app_warehouse
+                debug: true
+
             package:
-                scripts: package/*.sql
+                name: app_pkg
+                scripts:
+                    - shared_content.sql
```

Comment:
Even though package scripts are tested elsewhere, I wanted to make sure the syntax was correct (we're no longer accepting globs).